### PR TITLE
Update docs for cloud deployment using `helm`.

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
Thank you for your contribution @MUzairS15,
Let's discuss this on Websites's call on Monday. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

File: content/en/cloud/self-hosted/deploy-with-helm.md [hunk 1]
```diff
@@ -50,11 +50,16 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/cont
...
+Layer5 Cloud `postgres` instance requires [pg_cron](https://github.com/citusdata/pg_cron) extension to be enabled and configured in the `postgres` database to schedule PostgreSQL commands. 
+The cloud instance is bundled with required migrations to schedule jobs and execute required commands/sql queries.
+
+__NOTE: Configuring the extension in some other database will result in failure to apply SQL migrations.__
 
 ##### 1. Install Postgres database
 
 ```bash
-helm install -f ./install/postgresql/values.yaml postgres ./install/postgresql -n <namespace>
+helm repo add bitnami https://charts.bitnami.com/bitnami
```
Comment (leecalcote) [inline content/en/cloud/self-hosted/deploy-with-helm.md:61]: @MUzairS15, point of clarification - this change has nothing to do with pgcron, right? We just have the wrong instructions posted currently, right?

@sangramrath, FYI

Comment (leecalcote) [reply content/en/cloud/self-hosted/deploy-with-helm.md:61]: @sangramrath, should we be looking at using CloudNativePG here?

Comment (leecalcote) [reply content/en/cloud/self-hosted/deploy-with-helm.md:58]: @MUzairS15 can this be included as a sub-chart? Or would doing so hinder upgrades?

**Pull request merged**